<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Priority_Window</class>
 <widget class="QDialog" name="Priority_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Priority</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>241</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Primary</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Damage Group</string>
    </property>
   </widget>
   <widget class="QComboBox" name="damage_group_combo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="damage_group_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="phase_list">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>101</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="phase_combo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Phase</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>161</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Secondary</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance</string>
    </property>
   </widget>
   <widget class="QComboBox" name="secondary_combo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="secondary_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="add_prioirty_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_prioirty_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_prioirty_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up_prioirty_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down_prioirty_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>161</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Crew Type</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Crew Type</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="crew_type_list">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>141</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Priority_Window</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Priority_Window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
